\documentclass[parskip=full, numbers=noenddot]{scrreprt}

% language
\usepackage[english]{babel}
\usepackage[utf8]{inputenc}
\usepackage{csquotes}
\usepackage{url}
\usepackage[version=4]{mhchem}
\usepackage{siunitx}
\DeclareSIUnit\molar{\mole\per\cubic\deci\metre}
\DeclareSIUnit\Molar{\textsc{m}}
\DeclareSIUnit\calorie{cal}

% citations
\usepackage[
  natbib=true,
  backend=biber,
  doi=true,
  isbn=false,
  url=false,
  date=year,
  style=apa,
  citestyle=apa]{biblatex}
\addbibresource{dissertation.bib}
\usepackage{varioref}

% floats
\usepackage{graphicx}
  \graphicspath{ {./graphics/} }
\usepackage{subcaption}
\usepackage{tabularx}
  \newcolumntype{L}{>{\raggedright\arraybackslash}X}
\usepackage{booktabs}
\usepackage{pgfplotstable}
\usepackage{tikz}
\usetikzlibrary{shapes, arrows}
\tikzstyle{reagent} = [trapezium, trapezium left angle=70, trapezium right angle=110, text width=7em, text centered, draw=black, fill=blue!20]
\tikzstyle{process} = [rectangle, text width=7em, text centered, draw=black, fill=orange!30]
\tikzstyle{cloud} = [draw, ellipse]
%\tikzstyle{arrow} = [thick,->,>=stealth]
\tikzstyle{line} = [draw, -latex']

\title{Quantifying the bias in SELEX procedure}
\author{Candidate number XXXXX}

\begin{document}

%% NOTE ABOUT KMER PLOTS: currently I've added the PNG images so that the report isn't 60 MB.  Near the end I'll replace them with the corresponding PDF images.

\maketitle
% Will deal with the intricacies of the cover pages at the very end.

\begin{abstract}
% problem, approaches (in outline), major findings. make main points absolutely clear. aim for 200-250 words
  
Summary.
 
\end{abstract}

\tableofcontents

\chapter*{List of Abbreviations}
\label{ch:abbrev}

List of Abbreviations.

\chapter{EMSA-SELEX}
\label{ch:emsaselex}

\section{Introduction}
\label{sec:emsaselex_intro}

\subsection{Physiological importance of the nucleosome}
\label{ssec:emsaselex_intro_importance}

Eukaryotes package their DNA into chromatin through a series of coiling steps.  The fundamental repeating unit of chromatin is the nucleosome, which consists of a histone octamer with 147 base pairs (bp) of DNA wrapped around it.  Nucleosome positioning defines where nucleosomes are positioned within the genome \citep{struhl_determinants_2013}.  In contrast, nucleosome occupancy refers to the proportion of cells in a population that have a histone at a specific region in the genome. Nucleosome positioning and nucleosome occupancy are important determinants in gene expression because nucleosomes inhibit binding of DNA-binding proteins to DNA.  Specifically, nucleosome positioning inhibits transcription by preventing RNA polymerase from accessing chromatin, and prevent binding of transcription factors to regulatory elements.

\subsection{Nucleosome sequence preferences}
\label{ssec:emsaselex_intro_seqpref}

For a given 147-bp DNA sequence, the affinity of the histone octamer can vary over more than three orders of magnitude.  This is because the nucleotide sequence and the methylation state of nucleotides affect the physical flexibility of DNA, which is required for the wrapping of DNA around histone octamers.

The dinucleotide sequences AA, AT, and TA confer high flexbility to DNA.  Therefore, they are situated on the face of the helical repeat that directly interacts with the histone octamer, with a periodicity of approximately 10 bp \citep{struhl_determinants_2013}.  GC dinucleotides also occur periodically, but out of phase with the aforementioned dinucleotides.  These preferences were confirmed by a yeast-based genome-wide assay, which identified DNA regions stably wrapped in nucleosomes, and the associated dinucleotide probability distributions contributed to a computational model that confirmed these preferences \citep{segal_genomic_2006}.  Experiments with synthetic DNA in SELEX (systematic evolution of ligands by exponential enrichment) \citep{lowary_new_1998} have elucidated more rules for nucleosome positioning.  The `pentameric TG' sequence -- [(A or T)3NN(G or C)3NN]n -- has a very high affinity for the histone octamer, consistent with how AT-rich and GC-rich units confer flexibility of DNA.  `Phased TATA' sequences possessing multiple phased CTA trinucleotides also exhibit high affinity.

In contrast, the homopolymeric sequences poly(dA:dT) and poly(dG:dC) confer stiff structures that inhibit nucleosome formation.  These sequences are enriched in linker DNA between nucelosomes.  Poly(dA:dT) sequences are enriched in promoters, in particular that of \emph{Saccharomyces cerevisiae} \citep{struhl_determinants_2013}.  Depletion of nucleosomes at promoters on artificial chromosomes based on \emph{S. cerevisiae} genomic regions was shown to be dependent on the number and length of poly(dA:dT) sequences \citep{hughes_functional_2012}.

Additonally, DNA methylation disfavours nucleosome positioning \emph{in vivo}, as it increases the rigidity of DNA \citep{huff_dnmt1-independent_2014}.  Specifically, multiple tracts of methylated CpG sequences islands makes DNA have lower rise and twist along with higher roll values, thus making the DNA stiffer \citep{rao_systematic_2018, perez_impact_2012}.  However, the negative correlation between nucleosome occupancy and DNA methylation varies according to the genomic location.  For example, this negative correlation holds for CTCF regions, but not at promoters \citep{kelly_genome-wide_2012}.

\subsection{Purpose of study}
\label{ssec:emsaselex_intro_why}

To date, clean \emph{in vitro} sequence preference data for CpG methylation is not available.  This is in contrast to sequence preference data derived from MNase-seq on existing genomes in previous studies \citep{struhl_determinants_2013, segal_genomic_2006, huff_dnmt1-independent_2014}.  Furthermore, there has been no exploration into the effects of CH methylation (non-CpG methylation) on sequence preferences of nucleosomes.  Although CpG methylation is the predominant chemical modification of eukaryotic DNA, playing roles in gene regulation and physiology, CH methylation regulates development of stem cells and neurons \citep{guo_distribution_2014}.

SELEX is a promising method to identify the DNA sequences preferred for nucleosome positioning.  Unlike MNase-seq, SELEX employs a random DNA library, which has a higher complexity than genomic sequences and a more homogenous background sequence distribution.  \citet{lowary_new_1998} employed SELEX in uncovering sequence-based rules for nucleosome positioning, but analysis was limited by the technology at the time.  This project extended on their study by employing sequencing to a higher depth, allowing identification of signals with higher information content.

In this project, I aimed to study the sequence specificity of the nucleosome on methylated and non-methylated DNA.  I conducted four cycles of nucleosome SELEX using random DNA as the initial input library.  In each cycle, I reconstituted nucleosomes from \emph{Xenopus laevis} histone octamer and 147-bp DNA \citep{dyer_reconstitution_2003} methylated at various levels.  This included CpG methylation, methylation at all cytosines, and methylation at half the cytosines.  I extracted the DNA bound to nucleosomes at each cycle for sequencing, then assessed the distribution of mononucleotides and dinucleotides using Fourier transform \citep{lowary_new_1998, zhu_interaction_2018}.  Subsequently, the effect of histone octamer binding on the enrichment of mononucleotides in the sequencing libraries was investigated by producing 9-mer plots.

\section{Materials and Methods}
\label{sec:emsaselex_methods}

\subsection{DNA ligand design and preparation}
\label{ssec:emsaselex_methods_lig}

The sequence of the initial input library is 5$'$ GCTCTTCCGATCT nnnnnnnnnnnnnnnnnnnn AGATCGGAAGAGC 3$'$, where n denotes any nucleotide chosen at random. The input library DNA was made to 200 \SI{200}{\nano\Molar} in TE buffer with \SI{0.2}{\milli\Molar} EDTA.

PCR amplification was adapted from manufacturer's recommended protocols for Phusion High-Fidelity DNA Polymerase (ThermoFisher, cat no F530).  The sequence of the forward primer was 5$'$ CCCTACACGAC GCTCTTCC 3$'$, and the sequence of the reverse primer was 3$'$ GCCTTCTCG TGTGCAGAC 5$'$.  Thirteen cycles of PCR with recommended temperatures were followed by ten cycles with the \SI{98}{\celsius} denaturation temperature replaced with \SI{72}{\celsius}.  For the `half-C-methylated group', half of the dCTPs were replaced with 5-methylcytosine, and for the `all-C-methylated group', all of the dCTPs were replaced with 5-methylcytosine.   For CpG-methylation, methyltransferase (M.SssI) was added to DNA in conjunction with S-adenosylmethionine and \ce{MgCl2}, then incubated according to manufacturer's protocols.

\subsection{SELEX}
\label{ssec:emsaselex_methods_selex}

Nucleosome reconstitution procedures were adapted from \citet{dyer_reconstitution_2003} (`Reconstitution of Nucleosome Core Particles'), using \SI{2}{\Molar} \ce{KCl}.  To verify results, reconstituted nucleosomes were analysed by EMSA by using TBE 6\% DNA retardation gel and 0.2\% TBE buffer on ice.  Gel bands corresponding to unbound DNA and to reconstituted nucleosomes were sliced.  The sliced bands were dissolved in \SI{70}{\micro\litre} Tris pH 8.0, and then incubated at \SI{70}{\celsius} overnight.  Eluted DNA was amplified using procedures adapted from manufacturer's recommended protocols for Phusion High-Fidelity DNA Polymerase.  Here, 21 cycles of PCR with a \SI{98}{\celsius} denaturation temperature and a \SI{67}{\celsius} annealing temperature was followed by ten cycles with a \SI{79}{\celsius} denaturation temperature and a \SI{64}{\celsius} annealing temperature.

\subsection{Sequencing}
\label{ssec:emsaselex_methods_seq}

After SELEX, the eluted DNA was barcoded for sequencing.  Amplification was carried out using the procedures recommended for Phusion DNA polymerase for 29 PCR cycles, using \SI{5}{\micro\Molar} barcoded PE primer obtained from IDT.  All barcoded DNA was then pooled together and purified using 1.2x AMPure beads (Beckman Coulter) according to manufacturer's protocols.  Subsequently, the DNA was diluted to \SI{2}{\nano\Molar}.  Illumina HiSeq 4000 was then used for sequencing, following standard protocols, with $>$ 60 bp paired-end settings.  Raw sequences are demultiplexed, then the R1 and R2 reads of paired-end sequencing were merged according to \citet{zhu_interaction_2018}.

\subsection{Data analysis}
\label{ssec:emsaselex_methods_anal}

Fast Fourier transformation (FFT) was performed using the position along the sequence read as the time domain and the frequency of mononucleotides or dinucleotides at a specific position within the sequence as the frequency domain, following \citet{zhu_interaction_2018}.  From this, power spectra for mononucleotides and dinucleotides were obtained.  The phase of FFT was also examined at 0.102 per base pair.

\section{Results}
\label{sec:emsaselex_results}

\begin{figure}[htpb]
  \hspace*{-1cm}
\begin{tikzpicture}[node distance=2cm]
  %% nodes
  % input lib and amplification
  \node [reagent] (lig147) {lig147};
  \node [cloud, below of=lig147] (circle01) { };
  \node [process, below of=circle01] (amphalfmc) {Amplify with methyl-C + C};
  \node [process, left of=amphalfmc, xshift=-6cm] (amp) {Amplify};
  \node [process, right of=amphalfmc, xshift=2cm] (ampallmc) {Amplify with methyl-C};
  \node [reagent, below of=amp] (plain01) {Plain DNA};
  % amplified libraries (broadly)
  \node [reagent, below of=plain01] (plain02) {Plain DNA};
  \node [reagent, below of=amphalfmc, yshift=-2cm] (half-c-methylated) {Half-C-methylated};
  \node [reagent, below of=ampallmc, yshift=-2cm] (all-c-methylated) {All-C-methylated};
  \node [reagent, left of=half-c-methylated, xshift=-2cm] (cpg-methylated) {CpG-methylated};
  \node [process, above of=cpg-methylated, yshift=-0.75cm] (cpg) {CpG methylation};
  % nucleosome reconstitution
  \node [process, below of=half-c-methylated] (reconstitution) {Nucleosome reconstitution};
  % reconstituted nucleosomes
  \node [reagent, below of=reconstitution] (half-c-methylated-nuc) {Half-C-methylated nucleosome};
  \node [reagent, left of=half-c-methylated-nuc, xshift=-6cm] (plain-dna-nuc) {Plain DNA nucleosome};
  \node [reagent, left of=half-c-methylated-nuc, xshift=-2cm] (cpg-methylated-nuc) {CpG-methylated nucleosome};
  \node [reagent, right of=half-c-methylated-nuc, xshift=2cm] (all-c-methylated-nuc) {All-C-methylated nucleosome};
  % emsa
  \node [process, below of=half-c-methylated-nuc] (emsa) {EMSA and elute};
  % eluted
  \node [reagent, below of=emsa] (half-c-methylated-bub) {Half-C-methylated DNA bound and unbound};
  \node [reagent, left of=half-c-methylated-bub, xshift=-6cm] (plain-dna-bub) {Plain DNA bound and unbound};
  \node [reagent, left of=half-c-methylated-bub, xshift=-2cm] (cpg-methylated-bub) {CpG-methylated DNA bound and unbound};
  \node [reagent, right of=half-c-methylated-bub, xshift=2cm] (all-c-methylated-bub) {All-C-methylated DNA bound and unbound};
  % the bottom bit
  \node [cloud, below of=half-c-methylated-bub] (circle02) { };
  \node [process, below of=circle02] (barcoding) {Barcoding and sequencing};
  %% edges
  % lig147 to circle01
  \path [line] (lig147) -- (circle01);
  % between circle01 and amped libs
  \path [line] (circle01) --++ (0cm,-1cm) -| (amp);
  \path [line] (circle01) -- (amphalfmc);
  \path [line] (circle01) --++ (0cm,-1cm) -| (ampallmc);
  \path [line] (amp) -- (plain01);
  \path [line] (plain01) -| (cpg);
  \path [line] (plain01) -- (plain02);
  \path [line] (cpg) -- (cpg-methylated);
  \path [line] (amphalfmc) -- (half-c-methylated);
  \path [line] (ampallmc) -- (all-c-methylated);
  % amped libs to nucleosome reconstitution
  \path [line] (plain02) --++ (0cm, -1cm) -| (reconstitution);
  \path [line] (cpg-methylated) --++ (0cm, -1cm) -| (reconstitution);
  \path [line] (half-c-methylated) -- (reconstitution);
  \path [line] (all-c-methylated) --++ (0cm, -1cm) -| (reconstitution);
  % nucleosome reconstitution to nucleosomes
  \path [line] (reconstitution) --++ (0cm, -1cm) -| (plain-dna-nuc);
  \path [line] (reconstitution) --++ (0cm, -1cm) -| (cpg-methylated-nuc);
  \path [line] (reconstitution) -- (half-c-methylated-nuc);
  \path [line] (reconstitution) --++ (0cm, -1cm) -| (all-c-methylated-nuc);
  % nucleosomes to emsa
  \path [line] (plain-dna-nuc) --++ (0cm, -1cm) -| (emsa);
  \path [line] (cpg-methylated-nuc) --++ (0cm, -1cm) -| (emsa);
  \path [line] (half-c-methylated-nuc) -- (emsa);
  \path [line] (all-c-methylated-nuc) --++ (0cm, -1cm) -| (emsa);
  % emsa to eluteds
  \path [line] (emsa) --++ (0cm, -0.75cm) -| (plain-dna-bub);
  \path [line] (emsa) --++ (0cm, -0.75cm) -| (cpg-methylated-bub);
  \path [line] (emsa) -- (half-c-methylated-bub);
  \path [line] (emsa) --++ (0cm, -0.75cm) -| (all-c-methylated-bub);
  % eluteds to circle02
  \path [line] (plain-dna-bub) --++ (0cm, -1.5cm) -| (circle02);
  \path [line] (cpg-methylated-bub) --++ (0cm, -1.5cm) -| (circle02);
  \path [line] (half-c-methylated-bub) -- (circle02);
  \path [line] (all-c-methylated-bub) --++ (0cm, -1.5cm) -| (circle02);
  % bottom bit
  \path [line] (circle02) -- (barcoding);
  \path [line] (circle02) --++ (-11cm, 0cm) --++ (0cm, 16cm) -- (circle01) node[midway, above] {4 cycles};
\end{tikzpicture}
\caption{Overview of nucleosome EMSA-SELEX procedure}
\label{fig:selex}
\end{figure}

Four cycles of nucleosome EMSA SELEX (figure~\ref{fig:selex}) were carried out.  147-bp oligonucleotides with a 101-bp central random sequence -- termed lig147 \citep{zhu_interaction_2018} -- constituted the initial input library.  The experiment had four groups: plain (non-methylated) DNA, CpG-methylated DNA, DNA with all cytosines methylated (all-C-methylated DNA), and DNA with half of its cytosines methylated (half-C-methylated DNA).

\subsection{Nucleosome reconstitution}
\label{ssec:reconstnuc}

EMSA in all cycles yielded one strong band between 100 and 200 bp. It also yielded bands corresponding to reconstituted nucleosomes that became fainter as the proportion of histone octamer decreased in the solution (figure~\ref{fig:reconstnuc}).  DNA ligands that were not bound to nucleosomes and ligands that were bound to nucleosomes were eluted and then sequenced.

\begin{figure}[h]
  \centering
  \begin{subfigure}[htpb]{0.4\textwidth}
    \centering
    \includegraphics[width=\textwidth]{reconstnuc_a}
    \caption{Plain DNA}
    \label{fig:reconstnuc_a}
  \end{subfigure}
  \begin{subfigure}[htpb]{0.4\textwidth}
    \centering
    \includegraphics[width=\textwidth]{reconstnuc_b}
    \caption{Half-C-methylated DNA}
    \label{fig:reconstnuc_b}
  \end{subfigure}
  \begin{subfigure}[htpb]{0.4\textwidth}
    \centering
    \includegraphics[width=\textwidth]{reconstnuc_c}
    \caption{All-C-methylated DNA}
    \label{fig:reconstnuc_c}
  \end{subfigure}
  \begin{subfigure}[htpb]{0.4\textwidth}
    \centering
    \includegraphics[width=\textwidth]{reconstnuc_d}
    \caption{CpG-methylated DNA}
    \label{fig:reconstnuc_d}
  \end{subfigure}
  \caption{First-cycle EMSA in 6\% DNA retardation gels for the four experimental groups.  Lanes for all images correspond to these histone:octamer ratios (left to right): (1) 1.48:1, (2) 0.74:1, (3) 0.37:1, (4) 0.19:1, (5) 0.09:1, and (6) DNA without histone octamer added.  These results are representative for all cycles.}
  \label{fig:reconstnuc}
\end{figure}

The frequency of all mononucleotide and dinucleotide sequences at each position along the reads were calculated (figure~\ref{fig:freqplots}).  The data from the fourth cycle of EMSA-SELEX were used in all subsequent analyses because they exhibited the strongest signals.  All DNA libraries bound to nucleosomes showed a clear periodicity in mononucleotide and dinucleotide frequencies, but the periodicity in the plain DNA libraries was less clear.

\begin{figure}[htb]
  \centering
  \begin{subfigure}[htb]{0.6\textwidth}
    \centering
    \includegraphics[width=0.6\textwidth]{emsa_e8_counts}
    \caption{Plain DNA}
    \label{fig:freqplots_e8}
  \end{subfigure}
  \begin{subfigure}[htb]{0.6\textwidth}
    \centering
    \includegraphics[width=0.6\textwidth]{emsa_f8_counts}
    \caption{Half-C-methylated}
    \label{fig:freqplots_f8}
  \end{subfigure}
  \begin{subfigure}[htb]{0.6\textwidth}
    \centering
    \includegraphics[width=0.6\textwidth]{emsa_g8_counts}
    \caption{All-C-methylated}
    \label{fig:freqplots_g8}
  \end{subfigure}
  \begin{subfigure}[htb]{0.6\textwidth}
    \centering
    \includegraphics[width=0.6\textwidth]{emsa_h8_counts}
    \caption{CpG methylated}
    \label{fig:freqplots_h8}
  \end{subfigure}
  \caption{Frequencies of each mononucleotide and dinucleotide along the sequencing reads from libraries corresponding to 1:0.74 octamer:DNA ratios for the fourth cycle of EMSA-SELEX}
  \label{fig:freqplots}
\end{figure}

An additional library of plain DNA bound to nucleosomes prepared with additional rounds of AMPure bead purification to enrich the signals was analysed.  Characterisation of frequencies of mononucleotides at each position along the reads for this library yielded clear periodicities (figure~\ref{fig:enriched_counts}), which were confirmed to be approximately 10 bp by fast Fourier transform (figure~\ref{fig:enriched_power}).  However, this library exhibited biases towards G and T nucleotides.

\begin{figure}[htb]
  \centering
  \begin{subfigure}[htb]{0.8\textwidth}
    \centering
    \includegraphics[width=0.8\textwidth]{enriched-counts}
    \caption{}
    \label{fig:enriched_counts}
  \end{subfigure}
  \begin{subfigure}[htb]{0.8\textwidth}
    \centering
    \includegraphics[width=0.8\textwidth]{enriched-power}
    \caption{}
    \label{fig:enriched_power}
  \end{subfigure}
  \caption{Analysis of a library of plain DNA bound to nucleosomes enriched for signal. (a) shows the nucleotide counts as a function of position along the reads, and (b) shows the fast Fourier transform power spectra for the four nucleotides.}
  \label{fig:enriched}
\end{figure}

In contrast, all DNA libraries not bound to nucleosomes were depleted of this periodic signal.  This periodicity was assessed by fast Fourier transform, and the resulting power spectra showed that the periodicity was near 10 bp (figure~\ref{fig:powerspectra}).  Specifically, half-C-methylated and all-C-methylated DNA showed the strongest signals, % QUESTION: quantification of strength/height of peak?
CpG-methylated DNA showed signals of intermediate strength, and plain DNA showed the weakest signals.  Among the octamer-to-DNA concentrations tested, the 1:0.74 ratio gave rise to the strongest signals, and the signal intensity decreased as the proportion of DNA increased (data not shown).  Libraries corresponding to this octamer:DNA ratio were therefore used for all subsequent analyses.

\begin{figure}[htb]
  \centering
  \begin{subfigure}[htb]{0.6\textwidth}
    \centering
    \includegraphics[width=0.6\textwidth]{emsa_g7_power}
    \caption{DNA not bound to nucleosomes}
    \label{fig:powerspectra_unbound}
  \end{subfigure}
  \begin{subfigure}[htb]{0.6\textwidth}
    \centering
    \includegraphics[width=0.6\textwidth]{emsa_g8_power}
    \caption{DNA bound to nucleosomes}
    \label{fig:powerspectra_bound}
  \end{subfigure}
  \caption{Power spectra derived from the frequencies of all mononucleotide and dinucleotides, showing the effect of nucleosome binding.  Libraries from the all-C-methylated group in the fourth cycle of nucleosome EMSA-SELEX were shown here as representative spectra.  The 1:0.74 octamer:DNA ratio was chosen as it showed the strongest signals.}
  \label{fig:powerspectra}
\end{figure}

\subsection{Sequence preferences from fast Fourier transform analysis}
\label{ssec:nuseqpref_fft}

The phases and the amplitudes resulting from the fast Fourier transforms were then characterised, based on a frequency of 10.2 bp (figure~\ref{fig:radial_mc}).

When the cytosines were methylated, the amplitude for dinucleotides beginning with C increased, and their phases became more aligned.  Specifically, in the plain DNA library, the CA, CC, CG, and CT sequences were spread out over phases of \ang{80} to \ang{260} (figure~\ref{fig:radial_mc_plain}).  In the half-C-methylated and all-C-methylated libraries for the same octamer:DNA ratio, these sequences were spead out over phases of \ang{70} to \ang{140} (figures~\ref{fig:radial_mc_halfmc} and~\ref{fig:radial_mc_allmc}).  These dinucleotides became more aligned to TC.  The phases of the reverse complements of these dinucleotides -- namely GA, GC, and GG -- also became more aligned when the cytosines are methylated.   Using the AA sequence as a reference, the amplitudes for CT and CA were greater in the all-C-methylated library than in the half-C-methylated library.
% QUESTION: is there a better way to describe the `spread' of the CX sequences to replace eyeballing and saying `yeah, it looks like they're less spread alright'.

These effects were less pronounced for CpG methylation (figure~\ref{fig:radial_mc_cpg}).  Using the AA sequence as a reference, the amplitudes of CC and CT increased.  However, the CA, CC, CG, and CT sequences were still spread over a wide range of phases, covering \ang{80} to \ang{160}.

\begin{figure}[htb]
  \centering
  \begin{subfigure}[htb]{0.6\textwidth}
    \centering
    \includegraphics[width=0.6\textwidth]{emsa_e8_radial}
    \caption{Plain DNA}
    \label{fig:radial_mc_plain}
  \end{subfigure}
  \begin{subfigure}[htb]{0.6\textwidth}
    \centering
    \includegraphics[width=0.6\textwidth]{emsa_f8_radial}
    \caption{Half-C-methylated}
    \label{fig:radial_mc_halfmc}
  \end{subfigure}
  \begin{subfigure}[htb]{0.6\textwidth}
    \centering
    \includegraphics[width=0.6\textwidth]{emsa_g8_radial}
    \caption{All-C-methylated}
    \label{fig:radial_mc_allmc}
  \end{subfigure}
  \begin{subfigure}[htb]{0.6\textwidth}
    \centering
    \includegraphics[width=0.6\textwidth]{emsa_h8_radial}
    \caption{CpG methylated}
    \label{fig:radial_mc_cpg}
  \end{subfigure}
  \caption{Effects of cytosine methylation: radial plots showing amplitudes and phases for each dinucleotide}
  \label{fig:radial_mc}
\end{figure}

Certain dinucleotides containing cytosines were enriched when the cytosines were methylated, especially in CpG methylation (figure~\ref{fig:c2nt}).  In particular CC, CG, and GC were enriched in the CpG methylated group.  In constrast, CC was depleted in the all-C-methylated group.  However, in all samples, deviations from a 1:1 ratio were small.

% Use R to replace this with a dot plot later
\begin{figure}[htb]
  \centering
  \includegraphics[width=0.4\textwidth]{c2nt}
  \caption{Enrichment of dinucleotides in the 0.74:1 octamer:DNA ratio libraries for each experimental group.  The number of occurences of each dinucleotide in each library was calculated.  Dividing the count from the bound DNA library by the count from the unbound DNA library gave the ratios shown.} 
  \label{fig:c2nt}
\end{figure}

\subsection{Sequence preferences from sequence \emph{k}-mer counts}
\label{ssec:nuseqpref_kmer}

For each library, the frequency of each possible continuous 9-mer of nucleotides was counted.

Low correlation between bound and unbound libraries for CpG-methylated and half-C-methylated groups were observed (figure~\ref{fig:kmer_bound_cpg} and figure~\ref{fig:kmer_bound_half}), suggesting that bound and unbound libraries differ for these experimental groups.  Additonally, for these experimental groups, the nucleosome disfavours A/T-rich sequences, indicated by the A/T bias exhibited by the unbound libraries.  This effect seems to be stronger for the CpG-methylated group.  In contrast, there is a relatively high positive correlation between bound and unbound libraries for the all-C-methylated group (figure~\ref{fig:kmer_bound_all}).  This suggests that the bound and unbound libraries are similar, and that the effect of the nucleosome disfavouring A/T-rich sequences is weaker for all-C-methylated DNA.

However, both the bound and unbound all-C-methylated libraries -- relative to the corresponding bound or unbound plain DNA library -- exhibit the same patterns of bias (figure~\ref{fig:kmer_bound}).  More specifically, the biases are stronger for the bound libraries.  Bound and unbound libraries having the same bias in this case would likely be due to PCR bias (see section~\ref{ch:pcrbias}).

\begin{figure}[htb]
  \centering
  \begin{subfigure}[htb]{0.8\textwidth}
    \centering
    \includegraphics[width=0.8\textwidth]{kmer_CpGubXCpGb}
    \caption{CpG methylated, R = -0.402}
    \label{fig:kmer_bound_cpg}
  \end{subfigure}
  \begin{subfigure}[htb]{0.8\textwidth}
    \centering
    \includegraphics[width=0.8\textwidth]{kmer_halfubXhalfb}
    \caption{Half-C-methylated, R = -0.226}
    \label{fig:kmer_bound_half}
  \end{subfigure}
  \begin{subfigure}[htb]{0.8\textwidth}
    \centering
    \includegraphics[width=0.8\textwidth]{kmer_allubXallb}
    \caption{All-C-methylated, R = +0.710}
    \label{fig:kmer_bound_all}
  \end{subfigure}
  \caption{9-mer count scatter plots showing the effect of nucleosome reconstitution on each experimental group.  Each point represents a nucleotide 9-mer and the shading indicates the number of the respective nucleotide in the 9-mer.  9-mer counts were normalised against lig147 to give fold changes.  Data for plain DNA not shown as signal was too weak.}
  \label{fig:kmer_bound}
\end{figure}

\begin{figure}[htb]
  \centering
  \begin{subfigure}[htb]{0.8\textwidth}
    \centering
    \includegraphics[width=0.8\textwidth]{kmer_plainbXallb}
    \caption{Bound DNA, R = -0.162}
    \label{fig:kmer_bias_bound}
  \end{subfigure}
  \begin{subfigure}[htb]{0.8\textwidth}
    \centering
    \includegraphics[width=0.8\textwidth]{kmer_plainubXallub}
    \caption{Unbound DNA, R = -0.479}
    \label{fig:kmer_bias_unbound}
  \end{subfigure}
  \caption{9-mer count scatter plots comparing sequence biases.  Each point represents a nucleotide 9-mer and the shading indicates the number of the respective nucleotide in the 9-mer.  9-mer counts were normalised against lig147 to give fold changes.}
  \label{fig:kmer_bias}
\end{figure}

The frequency of continuous stretches of adenines (A-stretches) were also counted in each library (figure~\ref{fig:astretch}).  Methylation was associated with A-stretches being less represented in the libraries containing DNA bound to nucleosomes.  Compared to plain DNA, the half-C-methylated and all-C-methylated groups exhibited greater depletions in A-stretches.  This effect was more pronounced in the CpG methylated group.  Almost all of the groups exhibited ratios below 1.0.

% make this look more glamorous later by changing around the R code.  it's already informative
\begin{figure}[htb]
  \centering
  \includegraphics[width=0.7\textwidth]{astretch}
  \caption{Enrichment of continuous stretches of adenines for each experimental group.  The number of occurences of each k-mer of adenine stretches in each library was calculated (e.g. `5' corresponds to `AAAAA').  Dividing the count from the bound DNA library by the count from the unbound DNA library gave the ratios shown.}
  \label{fig:astretch}
\end{figure}

\section{Discussion}
\label{sec:emsaselex_discussion}

\subsection{Mechanisms leading to nucleosome sequence preferences}
\label{ssec:emsaselex_discussion_seqpref}

In nucleosome EMSA-SELEX, nucleosome binding introduced an approximately 10-bp periodicity in mononucleotide and dinucleotide frequencies.  The absence of nucleosome binding was associated with a depletion of this periodic signal.  These observations confirmed studies that suggest a $\sim$10-bp helical repeat and that dinucleotides tthat confer flexibility are positioned at the same periodicity \citep{struhl_determinants_2013}.  Furthermore, phases and amplitudes of the dinucleotides supported findings that indicate that TA occurs out of phase with GC.  In plain DNA (figure~\ref{fig:radial_mc_plain}) TA (\ang{20}) did not occur completely out of phase with GC (\ang{150}), but this is likely because the periodicity in this group was not as strong as the others (figure~\ref{fig:freqplots_e8}).

Stretches of adenines between five and nine nucleotides were depleted in libraries of DNA bound to nucleosomes, as indicated by normalised count ratios below 1.0 (figure~\ref{fig:astretch}).  This agrees with studies that reveal that poly(dA) tracts disfavour nucleosome positioning because they are stiff \citep{struhl_determinants_2013}.  Trends suggested that A-stretches were more depleted in methylated DNA than in plain DNA, and that A-stretches were more enriched if they were longer.  As the signals from plain DNA were weak, these conclusions may not be drawn.  Previous experiments with plain DNA libraries enriched for signal suggested that A-stretches were depleted as their length increases, concordant with previous theories regarding poly(dA) tracts.

In sum, the analyses in this project confirmed that nucleosome EMSA SELEX with current deep sequencing methods revealed the same conclusions as previous MNase-based assays.  % something about physiological relevance to 'broaden' the end of this section??

\subsection{Effect of methylation on nucleosome sequence preferences}
\label{ssec:emsaselex_discussion_methyl}

When cytosines were methylated, the amplitudes of CC, GG, and GC dinucleotides increased, and the phases of CC, CT, TC, and CA along with that of their reverse complements lined up more.  Methylation of cytosines makes DNA more rigid \citep{rao_systematic_2018, perez_impact_2012}.  So a possible explanation is that as more cytosines are methylated, rigid sequences are more strongly associated with phase angles that do not interfere with the bendability of DNA in such a way that would interfere with nucleosome positioning.  Another explanation is that methylated cytosines are structurally similar to thymines -- bulkier -- and therefore behaves more like thymines.  This explains TC and CC having similar phases, but does not explain why TT does not share this similarity.  Molecular simulations may be able to elucidate the structural basis of the observations.

Additionally, cytosine methylation was observed to be associated with enrichment of certain dinucleotides containing cytosines (figure~\ref{fig:c2nt}), but the effect size is small.  These small deviations from 1:1 ratios can be attributed to the low affinity of histone octamers to DNA, at least relative to transcription factors \citep{struhl_determinants_2013}. % probably find better citation to support this assertion, admittedly the sentence i was thinking of is not that strong
However, additional biological replicates must be done in order to evaluate the significance of the deviations from 1:1 ratios for each dinucleotide exhibited by each experimental group.

Thus these results provide preliminary insights into \emph{in vitro} sequence preference data for nucleosomes binding to CpG-methylated and CH-methylated DNA.  Further study could lead to better insights on how organisms use both forms of methylation to regulate developmental and physiological processes.  Such regulation may take advantage of how nucleosome positioning inhibits transcription.

From the \emph{k}-mer plots, for the CpG-methylated group, the nucleosome disfavours A/T-rich sequences, as suggested by the enrichment of A and T in the unbound libraries.  This nucleosome preference against A/T-rich sequences in CpG-methylated DNA is similar to how unmethylated DNA disfavours tracts of adenines and thymines \citep{struhl_determinants_2013}.  However, when half of or all of the cytosines were methylated, this effect of the nucleosome disfavouring A/T-rich sequences is diminished (figure~\ref{fig:kmer_bound_all}).  A potential explanation is that cytosine methylation make the DNA ligand apart from the A/T-rich tracts stiffer, and therefore different sections of the DNA ligand have `bendabilities' that are more similar.  As a result, the nucleosome exhibits sequence preferences that are less strong.

Investigating sequence enrichments in the all-C-methylated group relative to the plain DNA group revealed that similar biases were present in both unbound and bound DNA, suggesting that PCR accounted for this bias.  There is evidence to suggest that there are strong PCR biases in the presence of methylated cytosines.  Quantifying the sequence biases introduced by PCR would then allow removal of this bias to reveal the biases introduced by nucleosome sequence preferences.

\chapter{PCR Bias}
\label{ch:pcrbias}

\section{Introduction}
\label{sec:pcrbias_intro}

\subsection{Origin of PCR bias}
\label{ssec:pcrbias_intro_origin}

PCR is a known source of bias in massively parallel sequencing \citep{olova_comparison_2018}.  Among the processes in Illumina sequencing, PCR amplification during library preparation has been identified as the major source of bias.
% The corresponding figure from Aird et al. would be useful here
\citet{aird_analyzing_2011} measured frequencies of amplicons of varying GC content after each step in library preparation for Illumina sequencing, and found that DNA shearing, adapter ligation, gel size selection did not significantly introduce biases in response to GC content.  However, PCR depleted loci with GC content exceeding 65\% to the order of 1\% of the reference loci with intermediate GC content, and it also depleted loci with GC content lower than 12\% to the order of 10\% of the reference loci.  This pattern was observed even for 10 PCR cycles.

There are methods to mitigate this bias.  \citet{aird_analyzing_2011} found that this bias can be reduced by slowing the temperature ramping speed of thermocyclers, extending denaturation steps, and adding \SI{2}{\Molar} betaine.  Additionally, the choice of DNA polymerase enzyme has an effect.  Phusion DNA polymerase is commonly used because it has higher processivity and fidelity than most polymerases \citep{quail_optimal_2012}.  \citet{aird_analyzing_2011} also showed that replacing Phusion with AccuPrime Taq HiFi polymerase improved PCR bias.  Extension at \SI{60}{\celsius} using this enzyme retained more low-GC sequences, but also led to a decrease in yield of GC-rich reads.  \citet{quail_optimal_2012} showed that replacing Phusion with Kapa HiFi led to a more uniformm genome coverage.  However, this also led to a higher error rate, especially in TA-rich regions that Phusion often fails to amplify.

\subsection{How PCR bias affects reliability of sequencing results}
\label{ssec:pcrbias_intro_effects}

Commonly-used model organisms have GC content within the range in which bias is least likely introduced.  \emph{E. coli} and humans have genomes that have moderate GC content, with \emph{E. coli} having a GC content of 51\%.  PCR bias is still significant for these model organisms as they have important sequences that may be affected by PCR bias.  This includes single-nucleotide polymorphisms (SNPs), and GC-rich sequences like transcription start sites and first exons in eukaryotes.  %Additionally, humans exhibit `bad promoters'. % expand on the `bad promoters' data set from Ross et al.  Where did they get them, and what is their importance?

\subsection{Purpose of study}
\label{ssec:pcrbias_intro_why}

Although it is well-characterised that PCR introduces sequence biases, it is unknown which process within procedures that employ PCR amplification contributes the most to this bias, and to which degree.  Specifically, the biases introduced from the number of PCR cycles and from purification have not been characterised.  Such characterisation would be beneficial in developing a model to quantify PCR bias.  With such a model, these PCR biases can be removed in genomic studies to reveal true signals introduced by the factors tested in the relevant experiments.  PCR bias is especially relevant for nucleosome EMSA-SELEX employed in this project.  Sequence affinities for the nucleosome is weak, making the signal enrichment in the sequencing library weak, and makes it difficult to separate true signals introduced by nucleosome binding from biases introduced by PCR.  Both sequences bound to nucleosomes and sequences that are not exhibit the same PCR-introduced bias, rather than have biases that separate these two groups.

Here I assesed the sequence biases that could be introduced to the initial input library for nucleosome EMSA-SELEX.  I tested sequence biases introduced by the choice of DNA polymerase, choice of reagents, cycles of PCR, and purification of DNA after PCR.  From the resulting sequencing data, the distribution of k-mer nucleotides was assessed.

\section{Materials and Methods}
\label{sec:pcrbias_methods}

\subsection{DNA ligand design and preparation}
\label{ssec:pcrbias_methods_lig}

The same input library as \ref{ssec:emsaselex_methods_lig} was used, without methylation.

\subsection{Biases from enzymes}
\label{ssec:pcrbias_methods_enz}

PCR amplification followed manufacturers' specification for the following DNA polymerases: Phire Hot Start II DNA polymerase (ThermoFisher, F122S), Q5 Hot Start High-Fidelity DNA polymerase (New England BioLabs, M0493S), DreamTaq Hot Start DNA Polymerase (ThermoFisher, EP1701), and Phusion High-Fidelity DNA Polymerase (ThermoFisher, F530L).

For each enzyme, the input library was diluted by factors and then amplified for numbers of cycles as specified:

\begin{itemize}
  \item Dilute by a factor of $2^{4}$, then amplify for 8 cycles
  \item Dilute by a factor of $2^{8}$, then amplify for 12 cycles
  \item Dilute by a factor of $2^{12}$, then amplify for 16 cycles
  \item Dilute by a factor of $2^{16}$, then amplify for 20 cycles
  \item Dilute by a factor of $2^{20}$, then amplify for 24 cycles
\end{itemize}
    
To investigate bottleneck effects, the product from 24-cycle amplification was diluted by a factor of $2^{20}$.  From this, two samples were taken to be amplified using 24 cycles, and then the resulting amplification product was sequenced.

\subsection{Biases from PCR cycles}
\label{ssec:pcrbias_methods_pcr}

lig147 was diluted by a factor of $2^{20}$.  PCR amplification followed manufacturer's specification for Phusion High-Fidelity DNA Polymerase (ThermoFisher, F530L).  DNA was amplified for 4, 8, 12, 16, 20, 24, and 28 cycles, then re-diluted so that theoretical concentrations matched DNA that was amplified for 4 cycles.

\subsection{Biases from purification}
\label{ssec:pcrbias_methods_pur}

lig147 was purified using 1.2x Ampure beads (Agen Court AMPure) according to manufacturer's protocols, but with a \SI{15}{\micro\litre} elution volume.  Purification was repeated for 1, 2, 4, and 8 times.

\subsection{Biases from reagent vendors}
\label{ssec:pcrbias_methods_reagent}

lig147 was diluted by a factor of $2^{20}$.  It was then amplified for 24 cycles using manufacturer's specification for PCR amplification using Phusion High-Fidelity DNA Polymerase (ThermoFisher, F530L) and the reagents supplied by this vendor.

Combinations of Phusion High-Fidelity DNA Polymerase and dNTPs were investigated.  The polymerases used were from ThermoFisher (F530L) and New England BioLabs (M0530L).  The dNTPs used were from ThermoFisher (R0192), Agilent Technologies (200415-51), Kapa Biosystems (KN1009), and dNTPs containing 5-methylcytosine in place of dCTP from Zymo Research (D1030).  Twenty-four cycles of PCR using the eight combinations of polymerases and dNTPs proceeded according to manufacturer's specification for Phusion High-Fidelity DNA Polymerase (ThermoFisher, F530L).

\subsection{Sequencing}
\label{ssec:pcrbias_methods_seq}

The same procedures for sequencing as \ref{ssec:emsaselex_methods_seq} were used.

\subsection{Data analysis}
\label{ssec:pcrbias_methods_anal}

The same methods for fast Fourier transform and k-mer plot generation as \ref{ssec:emsaselex_methods_anal} were used.
% I can see this moved to results very easily
To investigate the effect of different vendors, a linear regression model was constructed.  The model used the frequencies of each nucleotide in the amplified library as the response variable.  The input variables were the enzyme vendor and dNTP vendor.

\section{Results}
\label{sec:pcrbias_results}

The initial input library lig147 was amplified and processed, with conditions varied to test biases introduced by polymerase enzymes, DNA purification, and reagent vendors.  After the PCR products were sequenced, the frequencies of each possible continuous 9-mer of nucleotides was counted.

% ORIGINAL
%% - Different oligonucleotide bias of different PCR enzymes
%% - The variance caused by bottleneck effect 
%% - Dependency of bias on PCR template concentration
%% - Bias caused by reagents from different company
%% - Bias from PCR Purification

\subsection{Biases from enzymes}
\label{ssec:pcrbias_result_enz}

Libraries amplified by Phusion, Phire, Dreamtaq, and Q5 DNA polymerase enzymes were analysed.  For each enzyme, the library from 24 cycles of PCR was compared to the library from 8 cycles of PCR (figure~\ref{fig:kmer_enz}).  All enzymes exhibited weak biases, as suggested by the relatively high correlation coefficients between 8 cycles and 24 cycles, indicating that the amplification products from these two groups were similar.  To be more precise, the enzymes all showed slight biases towards A and T, with Phusion exhibiting the weakest bias and Phire exhibiting the strongest bias.

\begin{figure}[htb]
  \centering
  \begin{subfigure}[htb]{0.5\textwidth}
    \centering
    \includegraphics[width=0.5\textwidth]{kmer_dreamtaq}
    \caption{DreamTaq, R = 0.709}
    \label{fig:kmer_enz_dreamtaq}
  \end{subfigure}
  \begin{subfigure}[htb]{0.5\textwidth}
    \centering
    \includegraphics[width=0.5\textwidth]{kmer_phire}
    \caption{Phire, R = 0.839}
    \label{fig:kmer_enz_phire}
  \end{subfigure}
  \begin{subfigure}[htb]{0.5\textwidth}
    \centering
    \includegraphics[width=0.5\textwidth]{kmer_q5}
    \caption{Q5, R = 0.760}
    \label{fig:kmer_enz_q5}
  \end{subfigure}
  \begin{subfigure}[htb]{0.5\textwidth}
    \centering
    \includegraphics[width=0.5\textwidth]{kmer_phusion}
    \caption{Phusion, R = 0.659}
    \label{fig:kmer_enz_phusion}
  \end{subfigure}
  \caption{9-mer count scatter plots comparing biases from different polymerase enzymes.  Each point represents a nucleotide 9-mer and the shading indicates the number of the respective nucleotide in the 9-mer.  9-mer counts were normalised against lig147 to give fold changes.}
  \label{fig:kmer_enz}
\end{figure}

\subsection{Biases from bottleneck effect}
\label{ssec:pcrbias_result_bn}

The effect of the bottleneck effect on PCR bias was then investigated.  In this context, the bottleneck effect refers to how some members of a large amplification library are likely to be lost by chance if a small sample of the library is taken for amplification.  For each enzyme investigated in subsection~\ref{ssec:pcrbias_result_enz}, the PCR product after 24 cycles diluted and sample from this dilution was amplified.  Comparing two samples for each enzyme suggested that bottleneck effects did not introduce large biases, as indicated by the strong correlations between the two samples for each enzyme (figure~\ref{fig:kmer_bn}).  However, Phire exhibited the greatest bias from the bottleneck effect, judging by the smallest correlation coefficient.

\begin{figure}[htb]
  \centering
  \begin{subfigure}[htb]{0.5\textwidth}
    \centering
    \includegraphics[width=0.5\textwidth]{kmer_dreamtaqBN}
    \caption{DreamTaq, R = 0.884}
    \label{fig:kmer_bn_dreamtaq}
  \end{subfigure}
  \begin{subfigure}[htb]{0.5\textwidth}
    \centering
    \includegraphics[width=0.5\textwidth]{kmer_phireBN}
    \caption{Phire, R = 0.697}
    \label{fig:kmer_bn_phire}
  \end{subfigure}
  \begin{subfigure}[htb]{0.5\textwidth}
    \centering
    \includegraphics[width=0.5\textwidth]{kmer_q5BN}
    \caption{Q5, R = 0.767}
    \label{fig:kmer_bn_q5}
  \end{subfigure}
  \begin{subfigure}[htb]{0.5\textwidth}
    \centering
    \includegraphics[width=0.5\textwidth]{kmer_phusionBN}
    \caption{Phusion, R = 0.844}
    \label{fig:kmer_bn_phusion}
  \end{subfigure}
  \caption{9-mer count scatter plots showing the bottleneck effect for different polymerase enzymes.  Each point represents a nucleotide 9-mer and the shading indicates the number of the respective nucleotide in the 9-mer.  9-mer counts were normalised against lig147 to give fold changes.}
  \label{fig:kmer_bn}
\end{figure}

\subsection{Biases from purification}
\label{ssec:pcrbias_result_pur}

Eight rounds of purification of the lig147 input library by AMPure beads did not introduce measurable nucleotide biases (figure~\ref{fig:kmer_pur}).

\begin{figure}[h]
  \centering
  \includegraphics[width=0.8\textwidth]{kmer_ampure}
  \caption{9-mer count scatter plot showing the effect of AMPure bead purification.  Each point represents a nucleotide 9-mer and the shading indicates the number of the respective nucleotide in the 9-mer.  9-mer counts were normalised against lig147 to give fold changes.}
  \label{fig:kmer_pur}
\end{figure}

\subsection{Biases from reagent vendors}
\label{ssec:pcrbias_result_reagent}

Different combinations of vendors for dNTPs and Phusion High-Fidelity DNA polymerases for PCR were investigated.  The fractions of each nucleotide after 24 rounds of amplification were calculated for each combination (figure~\ref{fig:linearmodel_nt}).

% I'm seriously considering changing these plots to dot plots and the vertical axes to be uniform around 0.20 to 0.30.  I have the data, not the code...
\begin{figure}[h]
  \centering
  \begin{subfigure}[h]{0.45\textwidth}
    \centering
    \includegraphics[width=0.45\textwidth]{linearmodel_a}
    \caption{adenine (A)}
    \label{fig:linearmodel_a}
  \end{subfigure}
  \begin{subfigure}[h]{0.45\textwidth}
    \centering
    \includegraphics[width=0.45\textwidth]{linearmodel_c}
    \caption{cytosine (C)}
    \label{fig:linearmodel_c}
  \end{subfigure}
  \begin{subfigure}[h]{0.45\textwidth}
    \centering
    \includegraphics[width=0.45\textwidth]{linearmodel_g}
    \caption{guanine (G)}
    \label{fig:linearmodel_g}
  \end{subfigure}
  \begin{subfigure}[h]{0.45\textwidth}
    \centering
    \includegraphics[width=0.45\textwidth]{linearmodel_t}
    \caption{thymine (T)}
    \label{fig:linearmodel_t}
  \end{subfigure}
  \caption{Fraction of each nucleotide after 24 cycles of amplification by each combination of dNTP and Phusion High-Fidelity DNA polymerase vendors.}
  \label{fig:linearmodel_nt}
\end{figure}

These fractions were then used to construct a linear model (equation~\ref{eqn:linearmodel}) to describe the effects of dNTP and Phusion High-Fidelity DNA polymerase vendor on the fraction of the nucleotides.

\begin{equation}
  \label{eqn:linearmodel}
  \textrm{fraction of nucleotide} \sim\ a (\textrm{enzyme vendor}) + b (\textrm{dNTP vendor}) + c
\end{equation}

\begin{table}[h]
  \centering
  \begin{tabular}{r}
     \\
    New England Biolabs\\
  \end{tabular}%
  \pgfplotstabletypeset[
  col sep = comma,
  ]{linearmodel_a.csv}  
  \caption{Coefficients for the linear model to describe the contribution of Phusion High-Fidelity DNA polymerase vendors to nucleotide fractions (`$a$' in equation~\ref{eqn:linearmodel}).  This is relative to the ThermoFisher High-Fidelity DNA polymerase.}
  \label{tab:linearmodel_coeffs_a}
\end{table}

\begin{table}[h]
  \centering
  \begin{tabular}{r}
     \\
    dNTP Kapa\\
    dNTP Agilent\\
    dNTP ZymoMeC\\
  \end{tabular}%
  \pgfplotstabletypeset[
  col sep = comma,
  ]{linearmodel_b.csv}  
  \caption{Coefficients for the linear model to describe the contribution of dNTP vendors to nucleotide fractions (`$b$' in equation~\ref{eqn:linearmodel}).  This is relative to the ThermoFisher dNTPs.}
  \label{tab:linearmodel_coeffs_b}
\end{table}


The model was then verified with experimental data (figure~\ref{fig:linearmodel_ver}).

% analyse the data and add how well the correlation is
\begin{figure}[h]
  \centering
  \includegraphics[width=0.4\textwidth]{linearmodel_plot}
  \caption{Verification of linear model. R = 0.9980243}
  \label{fig:linearmodel_ver}
\end{figure}

\section{Discussion}
\label{sec:pcrbias_discussion}
%% - Relevance of such bias to previous sequencing data
%% - Hints to data analysis in the future 
%% - A few discussions about Troubleshooting

All DNA polymerase enzymes tested -- Phusion, Phire, DreamTaq, and Q5  -- showed similar weak biases towards A and T.  Specifically, Phire showed the strongest bias, while Phusion showed the weakest bias.  The bottleneck effect of amplifying a small sample from a large input library was small, with Phire producing the largest bottleneck effect.  Biases introduced by AMPure bead purification and reagent vendors were minimal compared to biases from DNA polymerase enzymes.  As a result, PCR bias is most likely due to the reactions in PCR themselves rather than other procedures.  Furthermore, in the interests of preparing a sequencing library, Phire should be avoided as it introduces the greatest PCR biases.

However, the question of how much contribution A/T bias explains the variances observed in the \emph{k}-mer plots remains to be solved.  The initial lig147 input library has been shown to be biased towards G and T, likely because of the synthesis process, but this does not completely explain the variances.  To improve on these experiments, a better input library should be constructed.  This would lead to a better understanding of the biases introduced by nucleosome reconstitution and the use of different DNA polymerase enzymes.

An input library with improved filtering to remove sequences that have a tendency to form primer dimers can be used to investigate the effect of increased cycles of PCR on PCR bias.  The information obtained, supplementing data from this project, would be useful in creating a model to predict biases introduced by PCR.  PCR bias is at the same order as biases introduced by the weak affinity of the histone octamer towards specific DNA sequences.  An explicit PCR bias model would therefore be able to separate bias introduced by PCR and bias introduced by nucleosome positioning.

\chapter{Acknowledgements}
\label{ch:ack}

Acknowledgements.

\printbibliography

\end{document}